Pod/Classes/BONText.m

```diff
@@ -285,6 +285,16 @@ - (NSDictionary *)attributes
         attributes[NSParagraphStyleAttributeName] = paragraphStyle;
     }
 
+    // Text Underlining
+
+    if (self.underlineStyle != NSUnderlineStyleNone) {
```

Comment:
#nth I know `NSUnderlineStyleNone == 0` in the current implementation, and is unlikely to change, but I would feel better if this were set explicitly in `-init`.